kernel/comps/block/src/bio.rs

```diff
@@ -110,7 +110,7 @@ impl Bio {
     /// # Panics
     ///
     /// The caller must not submit a `Bio` more than once. Otherwise, a panic shall be triggered.
-    pub fn submit_sync(
+    pub fn submit_wait(
```

Comment:
The name `submit_and_wait` reads better and looks differently enough from `submit`.